src/main/java/com/amazon/opendistroforelasticsearch/knn/index/SpaceTypes.java

```diff
@@ -15,41 +15,52 @@
 
 package com.amazon.opendistroforelasticsearch.knn.index;
 
+import java.util.HashMap;
 import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
 
 /**
  * Enum contains space types for k-NN similarity search
```

Comment:
Could you update comment to describe why there are 2 values for each space type?